middleware/logger/utils.go

```diff
@@ -37,3 +41,48 @@ func statusColor(code int, colors fiber.Colors) string {
 		return colors.Red
 	}
 }
+
+type customLoggerWriter struct {
+	loggerInstance fiberlog.AllLogger
+	level          fiberlog.Level
+}
+
+func (cl *customLoggerWriter) Write(p []byte) (int, error) {
+	switch cl.level {
+	case fiberlog.LevelInfo:
```

Comment:
This order is wrong, it should be by `severity`:

- Trace
- Debug
- Info
- Warning
- Error
- Fatal

Reply:
shouldn't matter, but we can still customize it